corehq/apps/hqwebapp/static/hqwebapp/js/components/select_toggle.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Component for displaying a selection as a set of buttons.
+ * Creates an invisible <select> element to store the value and a set of buttons for the user to interact with.
+ *
+ * Required parameters
+ *  - options: Data to build select options. May be list of strings or list of objects with `id` and `text` properties.
```

Comment:
It likely makes sense for this to also accept an observable (same thing for the value param). I was lazy for now.